tools/prepare_python_system_tests.py

```diff
@@ -58,6 +44,22 @@
 zip_ref.close()
 print(zip_folder)
 
+src = os.path.join(zip_folder, os.listdir(zip_folder)[0], 'src')
```

Comment:
Avoid being inconsistent. You use `src = pathlib.Path().resolve().parents[1] / 'src'` in the other script.